Update README.md (#246)
callstack/react-native-viewpager: README.md

```

**Attention:** Note that you can only use `View` components as children of `ViewPager` (cf. folder */example*)

> For Android if View has own children set `collapsable=false` https://reactnative.dev/docs/view#collapsable otherwise react-native might remove this layout only view from hierarchy and it's children willl be rendered as separate pages

Reviewer: ```suggestion
. For Android if `View` has own children, set prop `collapsable` to false https://reactnative.dev/docs/view#collapsable, otherwise react-native might remove those children views and  and it's children will be rendered as separate pages
```
